### ISSUE TITLE
Shot props like bgImage, bgPosition don't work

### ISSUE THREAD

**Issue by dapetrov0** [jxnblk/styled-system#324]
There are shorthand props like bgImage, bgPosition, bgRepeat, bgSize in the table of styled functions. They don't work, maybe remove them from the table?

**Comment by aaronmcadam** [comment]
Yeah it looks like the implementation doesn't set aliases for these

**Comment by jxnblk** [comment]
Thanks! Yeah, all the aliases were removed in the latest version, looks like this was missed in the docs
